R/teal_data.R

```diff
@@ -26,45 +25,19 @@
 #'
 teal_data <- function(...,
                       join_keys = teal.data::join_keys(),
-                      code = character(0),
-                      check) {
+                      code = character(0)) {
   data_objects <- rlang::list2(...)
   if (inherits(join_keys, "join_key_set")) {
     join_keys <- teal.data::join_keys(join_keys)
   }
-  if (!missing(check)) {
-    lifecycle::deprecate_stop(
-      when = "0.4.0",
-      "teal_data(
-        check = 'check argument is deprecated. Use `verify()` to verify code reproducibility.
-        Find more information on https://github.com/insightsengineering/teal/discussions/945'
-      )"
-    )
-  }
```

Comment:
Just to be clear. You wrote

> The second commit is to deal with check argument: I've kept the documentation of verify() all the other related code and documentation has been removed.

We remove `check` parameter in `teal_data` and we keep `verify()` method. Can you confirm? Sorry for double checking, I can't understand the `all the other related code and documentation has been removed.` part of your sentence as it's part of the ` I've kept the documentation of verify()` sentence. Sounds like you kept documentation of `verify()` but removed code of `verify`

Reply:
Yes, I confirm that I deleted the check parameter but keep the verify method. I meant that I kept the verify() comment that was on the check argument and moved it to the code argument.